bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -497,6 +497,8 @@ <h4 class="join-firefox-title">{{ ftl('firefox-desktop-download-youve-already-go
 
 {% endblock %}
 
+{% include '/includes/sticky-promo.html' %}
```

Comment:
The sticky promo is still being inserted into the head before the CSS. It also seems to be causing some kind of markup invalidation:

<img width="1671" alt="image" src="https://user-images.githubusercontent.com/400117/98725352-eb990600-238c-11eb-9aee-912195f026f3.png">

I think this is what's causing IE9 to fail on the test run, as it renders this:

<img width="1676" alt="image" src="https://user-images.githubusercontent.com/400117/98725441-07041100-238d-11eb-9902-dd3145b7ced5.png">